simphony/io/h5_cuds.py

```diff
@@ -152,6 +155,47 @@ def add_mesh(self, mesh):
         self._file.flush()
         return m
 
+    def add_lattice(self, lattice, record=None):
```

Comment:
Yes, i think that this isn't part of this PR and that the discussion should take place somewhere else.